### DISCUSSION TITLE
Kort

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2345] [Annet]
Som et ledd i [inventory prosjektet](https://github.com/fremtind/jokul/projects/6) vårt ønsker vi tilbakemelding og stemmer fra både design og teknologi for å kartlegge hvilke komponenter og behov vi burde prioritere. Nedenfor følger en kort hypotese/slutning fra Designsystem-teamet, i tillegg til en rekke kommentarer vi har merket oss gjennom arbeidet.

Se på skjermbildene fra inventory [tilgjengelig her for referanse](https://www.figma.com/file/66kOJ5nCHjgtkfLW2sQWB8/Jøkul-Inventory-Innsiktsyntese?node-id=0%3A1). Legg gjerne til ønsker, behov, tanker, meninger og annet som kommenaterer i denne tråden, så vi kan sørge for at oppgavene blir så godt beriket med informasjon som mulig, dette gjør det enklere for alle involverte å ta tak i og løse de utfordringene vi møter sammen.

***

### TL;DR 👉 “Mangler et godt dokumentert kort i Jøkul”

- Veldig mange ulike kort: kort som er lenker, kort som er radio buttons, kort som er checkboxer,  kort som inneholder skjemaer, kort som har accordions, kort over bilder, tabellrader som blir kort på mobil
- Kan vi hente inspirasjon fra @mui/material for API? Paper-komponent med skygger, bygg kort ut fra den
- Vi mangler et godt dokumentert kort i Jøkul. Savnet fra flere team.
- Flere team har laget sine egne kort, få deler kode: Motor, PM, SeOpp, Boligselgerforsikring, Flyt
- Variasjon i skygger/elevasjon mellom kortene. Noen helt uten elevasjon.

**Comment by wkillerud** [comment]
[Evergreen](https://www.figma.com/community/file/974362403539471944) legger seg på et fint nivå også med elevasjon og ulike states, men ellers veldig lite. Gir stor fleksibilitet.

![image](https://user-images.githubusercontent.com/1223410/136389874-b020a602-2050-4ceb-95a5-331e1e20a697.png)

**Reply by onordbo** [reply]
Dette passer bra å se på i sammenheng med #1796. (cc @MonaHolm72)

**Comment by Stormoen** [comment]
Her er noen tanker vi har gjorde oss i sommer på kort. Både vanlige "inngangskort" og skjema/tabell "kort" vi bruker i motor og PM
https://www.figma.com/file/VoZ1tf59wAifHmjD0juUN7/VD-forum?node-id=1512%3A70909

![kort:skygger](https://user-images.githubusercontent.com/64519175/136942764-8a06612d-17c4-471c-92dc-d47c75027a4e.png)
.

**Comment by wkillerud** [comment]
Vi ser behovet for å gå grundigere til verks med kortene og se det i sammenheng med dokumentasjon av patterns. Selve "kort"-delen av kortet med farger og skygger er ikke teknisk komplisert, men vi må sørge for å løse de riktige problemene med den. Blir nok derfor å ta en slags designsprint på det som første del av "patterns-fasen" vi tenker å starte på nå i begynnelsen av november. Kommenter gjerne her om du vil være med på noe sånt! 🙋

**Reply by piofinn** [reply]
Starter litt håndsopprekking her, så får vi se om det blir noe snøballeffekt 😅

# 🙋‍♂️

**Reply by Slaine2** [reply]
vi har allerede laget et kart, men det er veldig rudimentært. Så jeg er gjerne med 🙋🏻‍♂️

**Reply by Stakset** [reply]
Jeg kan bli med hvis det er lite folk som melder seg. Hvor lang tid tenker dere?

**Reply by MonaHolm72** [reply]
Jeg vil være med

**Reply by karimarkhus** [reply]
Jeg vil også være med 🙋🏻‍♂️

**Reply by idalower-zz** [reply]
Kan også bli med

**Reply by kristineSorensen** [reply]
Hvis jeg har tid så vil jeg være med.

**Reply by karimarkhus** [reply]
jeg og @idalower har jobbet litt med kort til Flyt. Vi har laget en skisse, og ser for oss at vi kanskje trenger et kort som kan bygges ut med moduler. Sånn har google gjort det: https://www.figma.com/community/file/848275628252039675

![Screenshot 2021-11-04 at 16 16 52](https://user-images.githubusercontent.com/91268511/140345919-674d61c4-b7f4-4da3-81b0-4d8789f63e04.png)

**Comment by Slaine2** [comment]
Flott, so har vi en god gjeng. I begynnelsen kan vi bruke de 2 halvtimene i neste Forum til å utveksle informasjon om kravene.

**Comment by onordbo** [comment]
## Forum update (09/11/21)
Kjernegruppe for Kort-utvikling ble etablert; @Stakset, @karimarkhus, @piofinn og @kennidenni.

**Comment by onordbo** [comment]
Kort jobbes med videre i denne issuen: [https://github.com/fremtind/jokul/issues/2501](https://github.com/fremtind/jokul/issues/2501)

**Comment by onordbo** [comment]
Første versjon av kort er ute! https://github.com/fremtind/jokul/pull/2708 🚀
